source/getting-started/autostart-systemd.markdown

```diff
@@ -20,7 +20,7 @@ If the preceding command returns the string `systemd`, you are likely using `sys
 If you want Home Assistant to be launched automatically, an extra step is needed to setup `systemd`. You need a service file to control Home Assistant with `systemd`. If you are using a Raspberry Pi with Raspbian then replace the `[your user]` with `pi` otherwise use your user you want to run Home Assistant. `ExecStart` contains the path to `hass` and this may vary. Check with `whereis hass` for the location.
 
 ```bash
-$ su -c 'cat <<EOF >> /etc/systemd/system/home-assistant@[your user].service
+$ sudo su -c 'cat <<EOF >> /etc/systemd/system/home-assistant@[your user].service
```

Comment:
@bionutrient `sudo` is defiantly not specific to Raspbian or Linux for that part. It's a command that let's you execute a command with escalated privileges. 

The `su` command  when ran without specifying a user will default to the root user and is often called the superuser command(a bit misleading). 

In this case your escalating privileges to root to run another command as root.
I think(haven't tested) that you could replace the command in question with
```bash
$ sudo cat <<EOF >> /etc/systemd/system/home-assistant@[your user].service
```